OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListActivity.java

```diff
@@ -105,7 +105,7 @@ public void setTitle(int titleId) {
     public void setTitle(CharSequence title) {
         final View toolbar = findViewById(R.id.toolbar);
         if (toolbar != null) {
-            final View titleView = toolbar.findViewById(R.id.toolbar_title_view);
+            final View titleView = getTitleView();
             if (titleView != null && titleView instanceof TextView) {
```

Comment:
This `instanceof` in unnecessary as the return type of `getTitleView()` is `TextView`.